recipes/eNATL60/recipe.py

```diff
@@ -0,0 +1,55 @@
+from itertools import product
+
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+import pandas as pd
+
+
+regions = [1, 2, 3]
+season_months = {
+    'fma': pd.date_range("2010-02", "2010-05", freq="M"),
+    'aso': pd.date_range("2009-08", "2009-10", freq="M")
```

Comment:
@roxyboy, the missing timesteps you identified in https://github.com/pangeo-forge/staged-recipes/pull/24#issuecomment-929071973 appear to result from the fact that this `pd.date_range` is non-inclusive of the endpoint (i.e., October):

```python
In [1]: import pandas as pd

In [2]: dates = pd.date_range("2009-08", "2009-10", freq="M")

In [3]: len(dates)
Out[3]: 2

In [4]: dates
Out[4]: DatetimeIndex(['2009-08-31', '2009-09-30'], dtype='datetime64[ns]', freq='M')
```

So apparently the October data was never even cached, let alone written to the Zarr store.

To confirm, would you have expected a file with the following path to have existed at some point? 

```
url = (
    "https://ige-meom-opendap.univ-grenoble-alpes.fr/"
    "thredds/fileServer/meomopendap/extract/"
    "SWOT-Adac/Interior/eNATL60/Region03-interior-daily_2009-10.nc"
)
```

If so, that path does not appear to exist right now. Can that data be placed back on this server for us (for each of the three regions)?

Reply:
> the missing timesteps you identified in [#24 (comment)](https://github.com/pangeo-forge/staged-recipes/pull/24#issuecomment-929071973) appear to result from the fact that this `pd.date_range` is non-inclusive of the endpoint (i.e., October):

Ah, this must've been my error...

> If so, that path does not appear to exist right now. Can that data be placed back on this server for us (for each of the three regions)?

@cisaacstern The Opendap link should be working again now (fingers crossed).

Reply:
> The Opendap link should be working again now (fingers crossed).

Looks good. Caching the missing inputs to Google Cloud now.

Reply:
... turns out surface data for October are also missing (for the same reason). Caching those from the source server now as well.